win32/src/job_object_helper.c

```diff
@@ -1,34 +1,171 @@
-// Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
 #include <stdint.h>
 
+#include "windows.h"
+#include "processthreadsapi.h"
+#include "jobapi.h"
+
 #include "macro_utils/macro_utils.h"
 #include "c_logging/logger.h"
 
+#include "c_pal/gballoc_hl.h"
+#include "c_pal/gballoc_hl_redirect.h"
+#include "c_pal/thandle.h"
+
 #include "c_pal/job_object_helper.h"
 
+typedef struct JOB_OBJECT_HELPER_TAG {
+    HANDLE job_object;
+    DWORDLONG total_physical_memory;
```

Comment:
>DWORDLONG [](http://example.com/codeflow?start=4&length=9)

I think this is the first time i EVER see this data type. I thought it is typo the first time.